.github/workflows/unused-deps.yml

```diff
@@ -0,0 +1,38 @@
+# Runs a series of sanity checks for crate consumers.
+
+name: sanity
+
+on:
+  # we expect dependents to call this on a nightly basis
+  # schedule:
+  #  - cron: "0 0 * * *"
+  workflow_call: 
+    inputs:
+        # comma-separated list of features to check
+        features:
+          required: false
+          default: ""
+          type: string
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  unused-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      # udeps requires nightly
+      - uses: dtolnay/rust-toolchain@nightly
+      - uses: taiki-e/install-action@cargo-udeps
+      - name: Check for unused dependencies
+        run: cargo udeps --lib --features "${{ inputs.repository }}"
+      - uses: JasonEtco/create-an-issue@v2
```

Comment:
Should we standardize usage of this action over https://github.com/peter-evans/create-or-update-comment? Both seem workable but I like this one for the requirement of a formatted issue template

Reply:
I upgraded to this for precisely that reason. I'm up for upgrading to JasonEtco/create-an-issue across the board, but I haven't noticed any instances of peter-evans/create-or-update-comment in the present repo.